src/foundry/foundry.js/applications/formApplications/documentSheets/actorSheet.d.ts

```diff
@@ -1,210 +1,203 @@
+     * not meant to be used aside from being awaited.
+     */
+    protected _onDropActor(event: DragEvent, data: ActorSheet.DropData.Actor): Promise<unknown>;
 
-  /**
-   * Handle the final creation of dropped Item data on the Actor.
-   * This method is factored out to allow downstream classes the opportunity to override item creation behavior.
-   * @param itemData - The item data requested for creation
-   */
-  protected _onDropItemCreate(itemData: ActorSheet.OwnedItemData<O>): Promise<ActorSheet.OwnedItemData<O>>;
+    /**
+     * Handle dropping of an item reference or item data onto an Actor Sheet
+     * @param event - The concluding DragEvent which contains drop data
+     * @param data  - The data transfer extracted from the event
+     * @remarks This is intentionally typed to return `Promise<unknown>` to
+     * allow overriding methods to return whatever they want. The return type is
+     * not meant to be used aside from being awaited.
+     */
+    protected _onDropItem(event: DragEvent, data: ActorSheet.DropData.Item): Promise<unknown>;
 
-  /**
-   * Handle a drop event for an existing Owned Item to sort that item
-   */
-  protected _onSortItem(
-    event: DragEvent,
-    itemData: ActorSheet.OwnedItemData<O>
-  ): Promise<ActorSheet.OwnedItemData<O>> | undefined;
-}
+    /**
+     * Handle dropping of a Folder on an Actor Sheet.
+     * Currently supports dropping a Folder of Items to create all items as owned items.
+     * @param event - The concluding DragEvent which contains drop data
+     * @param data  - The data transfer extracted from the event
+     * @remarks This is intentionally typed to return `Promise<unknown>` to
+     * allow overriding methods to return whatever they want. The return type is
+     * not meant to be used aside from being awaited.
+     */
+    protected _onDropFolder(event: DragEvent, data: ActorSheet.DropData.Folder): Promise<unknown>;
 
-declare namespace ActorSheet {
-  type OwnedItemData<O extends Actor> = O extends Actor<any, infer I> ? I['data'] : never;
+    /**
+     * Handle the final creation of dropped Item data on the Actor.
+     * This method is factored out to allow downstream classes the opportunity to override item creation behavior.
+     * @param itemData - The item data requested for creation
+     */
+    protected _onDropItemCreate(
+      itemData: foundry.data.ItemData['_source'][] | foundry.data.ItemData['_source']
+    ): Promise<InstanceType<ConfiguredDocumentClass<typeof Item>>[]>;
 
-  /**
-   * @typeParam O - the type of the Entity which should be managed by this form
-   *                sheet
-   */
-  interface Data<O extends Actor = Actor> extends DocumentSheet.Data<O> {
-    actor: DocumentSheet.Data<O>['document'];
-    data: DocumentSheet.Data<O>['document']['data'];
-    items: DocumentSheet.Data<O>['document']['items'];
+    /**
+     * Handle a drop event for an existing Owned Item to sort that item
```

Comment:
I guess this is an old version of the doc text?

Reply:
yeah, must have overlooked that one. Updated it now.